src/Components/AffectedClustersTable/AffectedClustersTable.spec.ct.js

```diff
@@ -647,6 +705,7 @@ describe('empty failed affected clusters table', () => {
 
   it('renders table header', () => {
```

Comment:
I think we should start to use a TABLE_HEADERS constant here like https://github.com/RedHatInsights/ocp-advisor-frontend/blob/master/src/Components/ClustersListTable/ClustersListTable.spec.ct.js#L60